src/pybind/kaldi/table.py

```diff
@@ -342,6 +350,10 @@ class RandomAccessNnetChainExampleReader(_RandomAccessReaderBase,
     '''Random access table reader for nnet chain examples.'''
     pass
 
+class RandomAccessNnetExampleReader(_RandomAccessReaderBase,
```

Comment:
Line 661 in this file can also be removed.

Reply:
ok